packages/SwingSet/docs/configuration.md

```diff
@@ -103,6 +104,16 @@ The `bootstrap` property, if present, names the vat that is to be the
 bootstrap vat.  If this property is missing, there will be no
 bootstrap vat, which may be disallowed in certain use cases.
 
+The `pinBootstrap` property, if `true`, causes the bootstrap vat's root object's
+reference count to be artificially incremented by one as part of setting it up,
+so that the bootstrap vat won't be garbage collected even if its root object
+becomes nominally unreferenced.  The bootstrap vat will generally be garbage
+collected after it finishes executing the `bootstrap` method unless it formally
+exports its root object.  However, the host retains the ability to deliver
+messages to it directly, so if the host wants the bootstrap root to remain
+available to provide additional services after bootstrapping, this flag should
+be set.  Defaults to `false`.
+
```

Comment:
We should tighten this up. It increfs the root object, yes. But the goal is to keep the root *object* from being GCed (even if if becomes unreferenced). The *vat* never gets GCed: that would involve something like watching the c-list for exports and subscribed promises, and terminating the entire vat if that list ever became empty (because the vat could no longer receive messages from the outside world, so it could no longer acquire agency, so it might as well not exist) (except that exported promises would get rejected instead of remaining in limbo forever).

The second sentence needs to say something like "the root object will generally be collected unless it is referenced by some other object created during the `bootstrap()` method, or `bootstrap()` deliberately exports the root object".

Also the "to provide additional services" should mention "(e.g. `controller.queueToVatRoot`)".